src/vuex/modules/session.js

```diff
@@ -264,14 +264,16 @@ export default ({ apollo }) => {
       })
       return allSessionAddresses
     },
-    persistNotificationCounter(store, { notificationCounter }) {
-      localStorage.setItem(`notificationCounter`, notificationCounter)
-      state.notificationCounter = notificationCounter
+    persistNotificationAvailable(store, { notificationAvailable }) {
```

Comment:
we don't need to persist. we need to get the info from the server. the user could read the notification in another browser or on mobile.

Reply:
Yes, but what if the user refreshes the page. Or I'm not sure I follow you 100%. Maybe we should store `notificationAvailable` in DB, together with addressObjects?

Reply:
But yeah, good point with user reading it somewhere else. Although without DB I don't see a way to fix that

Reply:
instead we need the read status epic in. let's remove the code for persisting and then we can merge this in.